library/Makefile

```diff
@@ -21,6 +21,8 @@ endif
 # if were running on Windows build for Windows
 ifdef WINDOWS
 WINDOWS_BUILD=1
+else ifeq ($(shell uname -s),Darwin)
```

Comment:
Is this going to work for cross-compilation? Maybe we should qualify the test with a check that the compiler is cc, and not `cc` or `clang`?